hybridCarboniteMetadataService/src/test/scala/cromwell/services/metadata/hybridcarbonite/CarboniteWorkerActorSpec.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.services.metadata.hybridcarbonite
+
+import akka.actor.ActorRef
+import akka.testkit.{TestActorRef, TestProbe}
+import com.typesafe.config.ConfigFactory
+import common.validation.Validation._
+import cromwell.core.{TestKitSuite, WorkflowAborted, WorkflowFailed, WorkflowId, WorkflowSucceeded}
+import cromwell.services.metadata.MetadataArchiveStatus.Unarchived
+import cromwell.services.metadata.MetadataService.{QueryForWorkflowsMatchingParameters, QueryMetadata, WorkflowQueryResponse, WorkflowQueryResult, WorkflowQuerySuccess}
+import cromwell.services.metadata.WorkflowQueryKey._
+import cromwell.services.metadata.hybridcarbonite.CarboniteWorkerActor.CarboniteWorkflowComplete
+import cromwell.services.metadata.hybridcarbonite.CarbonitingMetadataFreezerActor.FreezeMetadata
+import org.scalatest.{FlatSpecLike, Matchers}
+
+import scala.concurrent.duration._
+
+class CarboniteWorkerActorSpec extends TestKitSuite("CarboniteWorkerActorSpec") with FlatSpecLike with Matchers {
+
+  val serviceRegistryActor = TestProbe()
+  val ioActor = TestProbe()
+  val carboniteFreezerActor = TestProbe()
+
+  val carboniterConfig = HybridCarboniteConfig.parseConfig(ConfigFactory.parseString(
+    """{
+      |   enabled = true
+      |   bucket = "carbonite-test-bucket"
+      |   filesystems {
+      |     gcs {
+      |       # A reference to the auth to use for storing and retrieving metadata:
+      |       auth = "application-default"
+      |     }
+      |   }
+      |}""".stripMargin
+  )).unsafe("Make config file")
+
+  val findWorkflowToCarboniteQuery: Seq[(String, String)] = Seq(
+    IncludeSubworkflows.name -> "false",
+    Status.name -> WorkflowSucceeded.toString,
+    Status.name -> WorkflowFailed.toString,
+    Status.name -> WorkflowAborted.toString,
+    MetadataArchiveStatus.name -> Unarchived.toString,
+    Page.name -> "1",
+    PageSize.name -> "1"
+  )
```

Comment:
Could you make this a constant value in an `object CarboniteWorkerActor` (rather than having to redefine it again here?)